mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
+
+Configure the site to use MVC
+========================================
+
+- Open the *project.json* file and add ``Microsoft.AspNet.Mvc`` to the ``dependencies`` property and the scripts section as highlighted below:
+
+.. literalinclude:: migratingfrommvc5/samples/WebApp1/src/WebApp1/project.json
+	:language: json
+	:emphasize-lines: 8-9, 29-31
+  
+``Microsoft.AspNet.StaticFiles`` is the static file handler. The ASP.NET runtime is modular, and you must explicitly opt in to serve static files.
 
-.. code-block:: javascript
+The ``scripts`` section is used to specify when specified build automation scripts should run. Visual Studio now has built-in support for running scripts before and after specific events. The ``scripts`` section above specifies `NPM <https://docs.npmjs.com/>`__, `Bower <http://bower.io/>`__ and `Gulp <http://gulpjs.com/>`__ scripts should run on the ``prepublish`` stage.  We'll talk about NPM, Bower, and Gulp later in the tutorial.
 
-	"dependencies": {
-		"Microsoft.AspNet.Server.IIS": "1.0.0-beta4",
-		"Microsoft.AspNet.Mvc": "6.0.0-beta4"
-	},
+For more information see `project.json <https://docs.asp.net/en/latest/conceptual-overview/understanding-aspnet5-apps.html#the-project-json-file>`_ and `Introducing .NET Core <http://docs.asp.net/en/latest/conceptual-overview/dotnetcore.html>`_.
 
-Now open Startup.cs and modify it as follows:
+- Open the *Startup.cs* file and change the code to match the following:
 
-.. code-block:: c#
+.. literalinclude:: migratingfrommvc5/samples/WebApp1/src/WebApp1/Startup.cs
+	:language: c#
+	:emphasize-lines: 6, 14-22
+	:linenos:  
+	:lines: 11-32
+	:dedent: 4
 
-	public void ConfigureServices(IServiceCollection services)
-	{
-		services.AddMvc();
-	}
 
-	public void Configure(IApplicationBuilder app)
-	{
-		app.UseMvc(routes =>
-		{
-			routes.MapRoute(
-				name: "default",
-				template: "{controller=Home}/{action=Index}/{id?}");
-		});
-	}
+``UseStaticFiles`` adds the static file handlers. As mentioned previously, the ASP.NET runtime is modular, and you must explicitly opt in to serve static files. For more information, see `Application Startup <http://docs.asp.net/en/latest/fundamentals/startup.html>`_ and `Routing <https://docs.asp.net/en/latest/fundamentals/routing.html>`__.
 
-At this point we are ready to create a simple Controller and View. Add a Controllers folder and a Views folder to the project. Add an MVC Controller called HomeController.cs class to the Controllers folder and a Home folder in the Views folder. Finally, add an Index.cshtml MVC View Page to the Views/Home folder. The project structure should be as shown:
+Add a controller and view
+========================================
+
+In this section you'll add a minimal controller and view to serve as place holders for the MVC 5 controller and views you'll migrate in the next section.
```

Comment:
"In this section" follows the same pattern as "In this topic" or "in this tutorial". Why is one okay and the other not?